### DISCUSSION TITLE
Exact time of a memo instead of elapsed time

### DISCUSSION THREAD

**Discussion by TLHD** [usememos/memos#4212] [Q&A]
Hi, guess it belongs here as I assume this is hiding somewhere and I just can't figure it out:
Is there any way to show the exact time a memo was made? Not the elapsed time. Like:"20:09 p. m"  instead of "6 hours ago".

**Comment by RoccoSmit** [comment]
There is no built in way to do this at this time, but see this comment for a workaround
https://github.com/usememos/memos/issues/3817#issuecomment-2300344224

**Reply by TLHD** [reply]
Will give that a try. Definitely good enough for me. Thanks for the fast reply!
